builder/qemu/step_prepare_efivars.go

```diff
@@ -0,0 +1,71 @@
+package qemu
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// stepPrepareEfivars copies the EFIVars file to the output, so we can boot
+// and use it as a RW flash drive
+type stepPrepareEfivars struct {
+	EFIEnabled  bool
+	OutputDir   string
+	EFIVarsPath string
+}
```

Comment:
nit: The EFI abbreviation is inconsistent and the field name `EFIVarsPath` has a bit of a stutter. 
```suggestion
type stepPrepareEFIVarsFile struct {
	EFIEnabled  bool
	OutputDir   string
	SourcePath string
}
```